spec/models/miq_queue_spec.rb

```diff
@@ -368,6 +368,19 @@
       expect(MiqQueue.get).to eq(nil)
     end
 
+    it "should skip putting message on queue in maintenance zone" do
+      Zone.seed
+
+      msg = MiqQueue.put(
+        :class_name  => 'MyClass',
+        :method_name => 'method1',
+        :args        => [1, 2],
+        :zone        => Zone.maintenance_zone&.name
+      )
+
+      expect(MiqQueue.get).to eq(nil)
```

Comment:
Should we just verify that the count is 0?

Reply:
ok, I just followed another expects written in this file